### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by CaledonianEH**
Updates the readme with new designs & more concise language. Also moves the boring licenses & legal notes to accordion menus.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,46 @@
...
+![OneConfig_banner](https://user-images.githubusercontent.com/62163840/193199250-6e28836d-cdb4-4967-9f6f-3f25e2dec19d.png)
 
-OneConfig is a next generation config library currently in development for all Minecraft mods.
 
-For a guide of how to use OneConfig please see our documentation at https://docs.polyfrost.cc
 
-![image](https://user-images.githubusercontent.com/67508414/172061081-7a32a139-d5c6-4f5a-907a-5af66deb799f.png)
-<br><br><br>
+# Introducing OneConfig
+Easy, stunning, expandable, open source. That sums it up quite well. We built OneConfig for the developer, to completly eliminate the hassle of making tools to configure their mods. Our API is elegant, and easy to learn. OneConfig was built from the ground up for this purpose. As for the user experience, it was our top priority. All of their mods will appear in one place, and they don't even need to install anything extra. The visual design is stunning, and is ready to grow whenever needed.
```
Comment (nxtdaydelivery) [inline README.md:6]: 'built oneconfig for the developer' and 'user experience was our top priority' clash a bit don't you think?

**Review by nxtdaydelivery — COMMENTED** [review]
looks good to me

**Comment by nxtdaydelivery** [comment]
on mobile at least, the padding on any of the dropdowns isn't great, but I don't think this is our issue/there is anything we can do about it.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,46 @@
...
+  <img src="https://user-images.githubusercontent.com/62163840/196335017-0ef66813-1500-4279-a6b1-565ec0741816.png">
+  <img src="https://user-images.githubusercontent.com/62163840/196335021-2df9bf19-d8d8-4872-ac57-1f6a53769cdd.png">
+  <img src="https://user-images.githubusercontent.com/62163840/196335012-9c0f098e-4214-42cf-a83a-9bcbe65a240b.png">
+</details>
 
-## License
+# How do I get it?
 
-OneConfig is free and open-source. Except where noted otherwise (below and/or in individual files), all code is licensed under BOTH 
+OneConfig Open Beta is now opened to the public! If you want to be the first to recieve news of new OneConfig updates, or new supported mods, check out our [**Discord server**](https://polyfrost.cc/discord). We're always in there so feel free to say hi!
```
Comment (xtrm-en) [inline README.md:21]: "Open beta is now opened", maybe change it to "OneConfig's open beta just released for all/everyone/just got out to the public" idk

File: README.md [hunk 3]
```diff
@@ -1,31 +1,46 @@
...
-For a guide of how to use OneConfig please see our documentation at https://docs.polyfrost.cc
 
-![image](https://user-images.githubusercontent.com/67508414/172061081-7a32a139-d5c6-4f5a-907a-5af66deb799f.png)
-<br><br><br>
+# Introducing OneConfig
+Easy, stunning, expandable, open source. That sums it up quite well. We built OneConfig for the developer, to completly eliminate the hassle of making tools to configure their mods. Our API is elegant, and easy to learn. OneConfig was built from the ground up for this purpose. As for the user experience, it was our top priority. All of their mods will appear in one place, and they don't even need to install anything extra. The visual design is stunning, and is ready to grow whenever needed.
 
-## I WANT IT!!!
+## Developer documentation
+For a guide of how to use OneConfig please see our wiki at https://docs.polyfrost.cc. It's very thorough, and actually written by the people who write OneConfig.
```
Comment (xtrm-en) [inline README.md:9]: "And ~~actually~~ written by the people who ~~write~~ develop OneConfig."?

**Review by xtrm-en — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pauliesnug) [reply README.md:6]: maybe like user experience and dveleoper experience are our top priority

**Review by Wyvest — APPROVED** [review]

**Review by xtrm-en — APPROVED** [review]
lgtm

**Pull request merged**